composer.json

```diff
@@ -23,5 +23,8 @@
         "psr-4": {
             "Ae\\FeatureBundle\\": ""
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.1"
```

Comment:
This bundle is compatible with PHP 5.4, so to prevent a dependency hell please add PHPUnit `^4.8` (afaik still maintained)